### PR TITLE
Adding bresenham circle program

### LINKED ISSUE #971: Bresenham Circle Algorithm in Python
This algo generates  circle .

### PR DESCRIPTION
**Pull request by robotjellyzone**
Fixes #971 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Added a Bresenham Circle Program In python using pygame 
#### Languages Used:
- Python

#### Files Added:
- Bresenham_Circle_Algorithm.py

### CODE REVIEW

File: Bresenham_Circle_Drawing_Algo/Bresenham_Circle_Algorithm.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
+# Install pygame library
+from pygame import gfxdraw
+
```
Comment (MastersAbh) [inline Bresenham_Circle_Drawing_Algo/Bresenham_Circle_Algorithm.py]: Remove this extra line

File: Bresenham_Circle_Drawing_Algo/Bresenham_Circle_Algorithm.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+import sys, pygame
+
```
Comment (MastersAbh) [inline Bresenham_Circle_Drawing_Algo/Bresenham_Circle_Algorithm.py]: 1 blank line is sufficient

**Comment by somya-kapoor** [comment]
@robotjellyzone Please correct the checklist in pr description. (it should be[x])

File: Bresenham_Circle_Drawing_Algo/Bresenham_Circle_Algorithm.py (hunk 1 above)
Comment (robotjellyzone) [reply Bresenham_Circle_Drawing_Algo/Bresenham_Circle_Algorithm.py]: But this is not extra line

Comment (robotjellyzone) [reply Bresenham_Circle_Drawing_Algo/Bresenham_Circle_Algorithm.py]: This is required for installation of pygame

Comment (MastersAbh) [reply Bresenham_Circle_Drawing_Algo/Bresenham_Circle_Algorithm.py]: Remove blank line no. 3

**Comment by jainaman224** [comment]
@MastersAbh Please review.

**Pull request merged**